File: Puzzles/Brent/reverse_words.rb
#DESCRIPTION:  Reverse the words in a sentence rather than all letters
#ALGORITHM:    Split the string into it's word components
#              Traverse the returned array in reverse order and joining
#              each word/element with a space between


def split_words(sentence)
	words = sentence.split.reverse.join(" ")
	puts words
	return words
end

split_words("A long long time ago")
#Output: ago time long long A

File: Puzzles/Completed/reverse_words.py
#DESCRIPTION:  Reverse the words in a sentence rather than all letters
#ALGORITHM:    Define a function called split_words that takes a string
#              as its input. The output is the inputs words put in reverse
#              order.
#Example: Input = 'A long long time ago' Output = 'ago time long long A'

def split_words(sentence):
	words=' '.join(string.split(sentence,' ')[::-1])
	return words

